rust/hello/webpack.config.js

```diff
@@ -86,14 +86,6 @@ module.exports = {
       template: path.join(__dirname, asset_entry),
       cache: false,
     }),
-    new CopyPlugin({
```

Comment:
Maybe we can also remove the `CopyPlugin` import on line 5 and uninstall the package using `npm uninstall --save copy-webpack-plugin`.

Reply:
good point. removed.